src/overlays/effects/ovl_Effect_Ss_Dt_Bubble/z_eff_ss_dt_bubble.h

```diff
@@ -3,6 +3,13 @@
 
 #include "global.h"
 
+typedef enum {
+    /* 0 */ DTBUBBLE_COLOR_PROFILE_0,
+    /* 1 */ DTBUBBLE_COLOR_PROFILE_1,
+    /* 2 */ DTBUBBLE_COLOR_PROFILE_2,
+    /* 3 */ DTBUBBLE_COLOR_PROFILE_3
+} DtBubbleColorProfile;
```

Comment:
I'm not really sure where we'll end up with this enum, boils down to whether each can be given a meaningful description.

Reply:
Possibly, I want to spawn each individually and see if there is a good way to describe them.

Reply:
So they do have distinct colors when spawned, so am just going the name the values now.